public/css/discussion.css

```diff
@@ -11,6 +11,7 @@
   border-width: 1px;
   border-color: lightgray;
   margin: 2px !important;
+  word-wrap:break-word;
```

Comment:
I know you didn't create the file but the indentation is different (this file is 2 spaces while the others have 4 space indentation)